.changeset/small-rules-vanish.md

```diff
@@ -0,0 +1,6 @@
+---
+"@redocly/openapi-core": patch
+"@redocly/cli": patch
+---
+
+Fixed false positive errors when validating media type examples that contain references.
```

Comment:
So that I understand this properly: we fixed a problem where no errors were reported - but the examples were wrong? So users might see errors without changing their API description, is that correct? Was it specifically when the changes went across files?

Reply:
>  So users might see errors without changing their API description, is that correct?

Actually vice versa. We show an error when there's no error (due to wrong references resolving). After the update, false errors should be gone.

Reply:
OK let's update the changelog entry to describe what the user might experience.